iHerb_autotests/iherb_product_details_page.py

```diff
@@ -0,0 +1,6 @@
+from selene import browser, by, query
+
+class IherbProductDetailsPage:
+
+    def get_product_reviews(self):
```

Comment:
does this return reviews, i.e. reviews text or reviews amount?

Reply:
Yes, it should return reviews text